staging/src/kubevirt.io/api/core/v1/types.go

```diff
@@ -2683,8 +2683,18 @@ type InterfaceBindingPlugin struct {
 	// If namespace is not specified, VMI namespace is assumed.
 	// version: 1alphav1
 	NetworkAttachmentDefinition string `json:"networkAttachmentDefinition,omitempty"`
+	// DomainAttachmentType is a standard domain definition that kubevirt supports.
+	// Supported values: "tap".
+	// The standard domain attachment can be used instead or in addition to the SidecarImage.
+	DomainAttachmentType DomainAttachmentType `json:"domainAttachmentType,omitempty"`
 }
 
+type DomainAttachmentType string
+
+const (
+	Tap DomainAttachmentType = "tap"
```

Comment:
I would add description about this binding method, that it use [libvirt generic network binding](https://libvirt.org/formatdomain.html#generic-ethernet-connection), maybe something along these lines:
```
Tap domain attachment type is a generic way to bind ethernet connection into guests using tap device 
https://libvirt.org/formatdomain.html#generic-ethernet-connection.
```